modules/ref-inference-endpoints.adoc

```diff
@@ -0,0 +1,220 @@
+
+:_module-type: REFERENCE
+
+[id='inference-endpoints_{context}']
+= Inference endpoints
+
+[role='_abstract']
+These examples show how to use inference endpoints to query the model.
+
+Caikit TGIS ServingRuntime for KServe::
+
+* `:443/api/v1/task/text-generation`
+* `:443/api/v1/task/server-streaming-text-generation`
+// * `:443/api/v1/task/text-classification`
+// * `:443/api/v1/task/token-classification`
+
+Caikit Standalone ServingRuntime for KServe::
+
+If you are serving multiple models, you can query `/info/models` or `:443 caikit.runtime.info.InfoService/GetModelsInfo` to view a list of served models.
++
+--
+.REST endpoints
+
+* `/api/v1/task/embedding`
+* `/api/v1/task/embedding-tasks`
+* `/api/v1/task/sentence-similarity`
+* `/api/v1/task/sentence-similarity-tasks`
+* `/api/v1/task/rerank`
+* `/api/v1/task/rerank-tasks`
+* `/info/models`
+* `/info/version`
+* `/info/runtime`
+--
++
+--
+.gRPC endpoints
+
+* `:443 caikit.runtime.Nlp.NlpService/EmbeddingTaskPredict`
+* `:443 caikit.runtime.Nlp.NlpService/EmbeddingTasksPredict`
+* `:443 caikit.runtime.Nlp.NlpService/SentenceSimilarityTaskPredict`
+* `:443 caikit.runtime.Nlp.NlpService/SentenceSimilarityTasksPredict`
+* `:443 caikit.runtime.Nlp.NlpService/RerankTaskPredict`
+* `:443 caikit.runtime.Nlp.NlpService/RerankTasksPredict`
+* `:443 caikit.runtime.info.InfoService/GetModelsInfo`
+* `:443 caikit.runtime.info.InfoService/GetRuntimeInfo`
+--
+
+ifdef::upstream[]
+[NOTE]
+--
+By default, the Caikit Standalone Runtime exposes REST endpoints for use. To use gRPC protocol, manually deploy a custom Caikit Standalone ServingRuntime. For more information, see link:{odhdocshome}/serving-models/#adding-a-custom-model-serving-runtime-for-the-single-model-serving-platform_serving-large-models[Adding a custom model-serving runtime for the single-model serving platform]. 
```

Comment:
Line 51, second sentence. Are we missing the word "the" before gRPC protocol, or is it just the way I'm reading it?

Reply:
I'm treating this the same way as "REST" since they're both protocol. If we should add "the" before gRPC, we should also add "the" before "REST." Does that make sense? Does that change anything for you here?